lib/postgrex/builtins.ex

```diff
@@ -190,7 +190,7 @@ end
 
 defmodule Postgrex.Lexeme do
   @moduledoc """
-  Struct for Postgres Lexeme (A Tsvector type is composed of multiple lexemes)
+  Struct for PostgreSQL lexeme (A tsvector type is composed of multiple lexemes)
```

Comment:
`lexeme` and `tsvector` are actually type name in lower case. 

Maybe we can wrap them with backticks (e.g.  `` `lexeme` ``)?

Reply:
Backticks sounds like a good call. ALso, please rebase!

Reply:
Added backticks, and rebased!